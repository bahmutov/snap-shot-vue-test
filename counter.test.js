import Vue from 'vue'
import Counter from './counter'
const renderer = require('vue-server-renderer').createRenderer()

test('counter', () => {
  const vm = new Vue({
    el: document.createElement('div'),
    render: (h) => h(Counter),
  })
  renderer.renderToString(vm, (err, str) => {
    expect(str).toMatchSnapshot();
  })
})
