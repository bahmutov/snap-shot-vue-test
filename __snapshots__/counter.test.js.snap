exports[`test counter 1`] = `
"<div server-rendered=\"true\">
    counter: 0
  </div>"
`;
